### DISCUSSION TITLE
A version of AIDente that simply puts Mac into "Optimised battery charging" mode

### DISCUSSION THREAD

**Discussion by arazabishov** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#616] [Ideas]
Hi there! 

I have recently posted a [question](https://www.reddit.com/r/MacOS/comments/t7xaqk/q_how_to_force_optimised_battery_charging_mode/) to Reddit asking about tools that would help me to preserve the battery life on Mac. I want to minimise the amount of tempering involved with the power management system (for obvious reasons mentioned in the disclaimer). 

Now the idea: would you be interested in building a new app that simply "forces" Mac into the "Optimised charging state"? This way we can still rely on macOS smarts for battery management. 

Another way of looking at: the app is simply a way to give the OS a hint that Mac will be sitting on charged for a long time, so it is better to be in the "Optimised battery charging" mode :)

With this solution, I will have a peace of mind that the tool does not break some bespoke invariant for power management that Apple engineers have considered. It will simply help to guide the macOS into the right power management mode.

**Comment by Bachsau** [comment]
This is mostly what AlDente does. Both systems make use of the same control mechanisms, with the only difference that Apple's optimized charging feature doesn't put the user in control. There is no additional "tampering" involved.

**Comment by vladcudoidem** [comment]
Hi @arazabishov, did you end up using Al Dente?

**Comment by arazabishov** [comment]
@vladcudoidem nope, I decided to rely on OS for power/battery management.

**Reply by vladcudoidem** [reply]
Has that worked out well for you?

**Reply by arazabishov** [reply]
Yup. The battery will age regardless of what you do, so I thought might as well not worry about it.
